### DISCUSSION TITLE
Unable to load OMF file

### DISCUSSION THREAD

**Discussion by m-sb** [pyvista/pyvista#5809] [Q&A]
### First check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the PyVista documentation, with the integrated search.
- [X] I already searched in Google "How to X in PyVista" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I will NOT make duplicate discussions or issues in the future.

### Commit to Help

- [X] I commit to help with one of those options.

### Sample Code What is the problem, question, or error?
Write a short description telling me what you are doing, what you expect to happen, and what is currently happening.

```python
Unable to open omf file. Getting error as module 'omf' has no attribute 'OMFReader'
```

### Description

I am trying to load the omf file. and getting error.
[omf_file.zip](https://github.com/pyvista/pyvista/files/14694190/omf_file.zip)

### System Information

```shell
--------------------------------------------------------------------------------
  Date: Thu Mar 21 16:46:57 2024 India Standard Time

                OS : Windows
            CPU(s) : 12
           Machine : AMD64
      Architecture : 64bit
               RAM : 32.0 GiB
       Environment : Python
        GPU Vendor : NVIDIA Corporation
      GPU Renderer : NVIDIA A40-4Q/PCI/SSE2
       GPU Version : 4.5.0 NVIDIA 472.98
  MathText Support : False

  Python 3.12.2 (tags/v3.12.2:6abddd9, Feb  6 2024, 21:26:36) [MSC v.1937 64
  bit (AMD64)]

           pyvista : 0.43.4
               vtk : 9.3.0
             numpy : 1.26.4
        matplotlib : 3.8.3
            scooby : 0.9.2
             pooch : 1.8.1
            pillow : 10.2.0
           IPython : 8.22.2
        ipywidgets : 8.1.2
        jupyterlab : 4.1.5
      nest_asyncio : 1.6.0
--------------------------------------------------------------------------------
```

### Screenshots

![image](https://github.com/pyvista/pyvista/assets/22414477/991eded6-a7b2-410d-8e91-7cef2dd098a9)

**Comment by tkoyama010** [comment]
@banesullivan Any idea for this?

**Reply by banesullivan** [reply]
Would you please include which version of omf you have installed?

**Reply by banesullivan** [reply]
Do you perhaps have a folder named "omf" under your working directory that could be conflicting with the import?
